python/lsst/afw/image/mask.py

```diff
@@ -1,7 +1,17 @@
 from __future__ import absolute_import
+from future.utils import with_metaclass
+
+from abc import ABCMeta
 
 from ._mask import *
 from .slicing import supportSlicing
 
 supportSlicing(MaskU)
 
+class Mask(with_metaclass(ABCMeta, object)):
+    pass
+
+for cls in (MaskU, ):
+    Mask.register(cls)
+    supportSlicing(cls)
+
```

Comment:
No, the reason was to introduce the ABC `Mask` (in concert with `Image`, `MaskedImage` and `Exposure`) to be used for `isinstance` testing. Instead of using a regexp to test against the output of `__repr__` which is problematic and ugly.